lib/pages/class_feedback/view/classes_feedback_checklist.dart

```diff
@@ -0,0 +1,177 @@
+  }
+
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    fetchFeedbackProvided();
+  }
+
+  Future<Map<String, dynamic>> fetchFeedbackProvided() async {
+    final feedbackProvider =
+        Provider.of<FeedbackProvider>(context, listen: true);
+    final authProvider = Provider.of<AuthProvider>(context, listen: false);
+    await feedbackProvider
+        .getProvidedFeedbackClasses(authProvider.uid)
+        .then((value) => setState(() => classesFeedback = value));
+    return classesFeedback;
+  }
+
+  Widget checklistPage() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        const SizedBox(height: 12),
+        Text(
+          S.current.sectionFeedbackTODO,
+          style: Theme.of(context).textTheme.headline6,
+        ),
+        const SizedBox(height: 6),
+        FeedbackClassList(
+          classes: widget.classes
+              .where((element) => !classesFeedback.containsKey(element.id))
+              .toSet(),
+          done: false,
+        ),
+        const Divider(thickness: 4),
+        const SizedBox(height: 12),
+        Text(
+          S.current.sectionFeedbackCompleted,
+          style: Theme.of(context).textTheme.headline6,
+        ),
+        const SizedBox(height: 6),
+        FeedbackClassList(
+          classes: widget.classes
+              .where((element) => classesFeedback.containsKey(element.id))
+              .toSet(),
+          done: true,
+        ),
+      ],
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return AppScaffold(
+      title: Text(S.current.navigationClassesFeedbackChecklist),
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 12),
+          child: Column(
+            children: [
+              if (widget.classes != null)
+                checklistPage()
+              else
+                ErrorPage(
+                  errorMessage: S.current.messageNoClassesYet,
+                  imgPath: 'assets/illustrations/undraw_empty.png',
+                  info: [
+                    TextSpan(text: '${S.current.messageGetStartedByPressing} '),
+                    WidgetSpan(
+                      alignment: PlaceholderAlignment.middle,
+                      child: Icon(
+                        Icons.edit_outlined,
+                        size:
+                            Theme.of(context).textTheme.subtitle1.fontSize + 2,
+                      ),
+                    ),
+                    TextSpan(text: ' ${S.current.messageButtonAbove}.'),
+                  ],
+                ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class FeedbackClassList extends StatelessWidget {
+  const FeedbackClassList({Key key, this.classes, this.done}) : super(key: key);
+  final Set<ClassHeader> classes;
+  final bool done;
+
+  Widget buildFeedbackClassList(ClassHeader header) => Column(
```

Comment:
Just inline this method where you use it (`classes.map((classHeader) => Column(...))`, no point in naming it